### PR TITLE
Compact widget-tops on smaller laptops, but not tablets.

### PR DESCRIPTION
**Pull request by MichaelArestad**
A wee bit more compact on screens sizes less than 700px tall and more than 981 wide.

Should be cool. Keeps touch target size in tact for tablets/smaller. Might need adjusting a little.

### CODE REVIEW

**Comment by westonruter** [comment]
customize_twenty_thirteen__wordpress_and_customize_twenty_thirteen__wordpress_and_widget-customizer_php_-_wordpress-develop_-____projects_vvv_www_wordpress-develop_-4

File: widget-customizer.css [hunk 1]
```diff
@@ -589,4 +589,42 @@ body.adding-widget #customize-preview {
...
\ No newline at end of file
+#available-widgets [class*="twitter"] .widget-title:before { content: "\f301"; }
+
+
+@media screen and (max-height: 700px) and (min-width: 981px) {
+	.customize-control {
+		margin-bottom: 0;
+	}
+	.widget-top {
+		border-none;
```
Comment (westonruter) [inline widget-customizer.css:600]: @MichaelArestad I just noticed this. It's a syntax error. Should it be `border: none`?

Comment (MichaelArestad) [inline widget-customizer.css:600]: @westonruter No. That whole rule shouldn't be there. Not needed.

Comment (westonruter) [inline widget-customizer.css:600]: So we should remove all of `.widget-top { … }` or just this property?

Comment (MichaelArestad) [inline widget-customizer.css:600]: Just that property.

Comment (westonruter) [inline widget-customizer.css:600]: Done in cff88a1

**Pull request merged**